pkg/controller/expcertificates/internal/test/test.go

```diff
@@ -296,3 +317,43 @@ func generateCSRImpl(crt *cmapi.Certificate, pk []byte) ([]byte, error) {
 
 	return csrPEM, nil
 }
+
+func generateLocallySignedTemporaryCertificate(crt *cmapi.Certificate, pkData []byte) ([]byte, error) {
```

Comment:
This function seems to be identical to the one in the issuing controller from what I can see?